demos/fleet_provisioning/fleet_provisioning_csr/demo_config.h

```diff
@@ -0,0 +1,215 @@
+ * 2. Define LIBRARY_LOG_NAME and  LIBRARY_LOG_LEVEL.
+ * 3. Include the header file "logging_stack.h".
+ */
+
+/* Include header that defines log levels. */
+#include "logging_levels.h"
+
+/* Logging configuration for the Demo. */
+#ifndef LIBRARY_LOG_NAME
+    #define LIBRARY_LOG_NAME    "FLEET_PROVISIONING_DEMO"
+#endif
+
+#ifndef LIBRARY_LOG_LEVEL
+    #define LIBRARY_LOG_LEVEL    LOG_INFO
+#endif
+
+#include "logging_stack.h"
+
+/************ End of logging configuration ****************/
+
+/**
+ * @brief Details of the MQTT broker to connect to.
+ *
+ * This is the Thing's Rest API Endpoint for AWS IoT.
+ *
+ * @note Your AWS IoT Core endpoint can be found in the AWS IoT console under
+ * Settings/Custom Endpoint, or using the describe-endpoint API.
+ *
+ * #define AWS_IOT_ENDPOINT               "...insert here..."
+ */
+
+/**
+ * @brief AWS IoT MQTT broker port number.
+ *
+ * In general, port 8883 is for secured MQTT connections.
+ *
+ * @note Port 443 requires use of the ALPN TLS extension with the ALPN protocol
+ * name. When using port 8883, ALPN is not required.
+ */
+#define AWS_MQTT_PORT    ( 8883 )
+
+/**
+ * @brief Path of the file containing the server's root CA certificate.
+ *
+ * This certificate is used to identify the AWS IoT server and is publicly
+ * available. Refer to the AWS documentation available in the link below
+ * https://docs.aws.amazon.com/iot/latest/developerguide/server-authentication.html#server-authentication-certs
+ *
+ * Amazon's root CA certificate is automatically downloaded to the certificates
+ * directory from @ref https://www.amazontrust.com/repository/AmazonRootCA1.pem
+ * using the CMake build system.
+ *
+ * @note This certificate should be PEM-encoded.
+ * @note This path is relative from the demo binary created. Update
+ * ROOT_CA_CERT_PATH to the absolute path if this demo is executed from elsewhere.
+ */
+#ifndef ROOT_CA_CERT_PATH
+    #define ROOT_CA_CERT_PATH    "certificates/AmazonRootCA1.crt"
+#endif
+
+/**
+ * @brief Path the file containing the provisioning claim certificate. This
+ * certificate is used to authenticate with AWS IoT for provisioning by claim.
+ *
+ * For information about provisioning by claim, see the following AWS documentation:
+ * https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html#claim-based
+ *
+ * @note This certificate should be PEM-encoded.
+ *
+ * #define CLAIM_CERT_PATH    "...insert here..."
+ */
+
+/**
+ * @brief Path the file containing the provisioning claim private key. This
+ * key corresponds to the provisioning claim certificate and is used to
+ * authenticate with AWS IoT for provisioning by claim.
+ *
+ * For information about provisioning by claim, see the following AWS documentation:
+ * https://docs.aws.amazon.com/iot/latest/developerguide/provision-wo-cert.html#claim-based
+ *
+ * @note This private key should be PEM-encoded.
+ *
+ * #define CLAIM_KEY_PATH    "...insert here..."
```

Comment:
Mentioning "private" avoids ambiguity of the key type.
```suggestion
 * #define CLAIM_PRIVATE_KEY_PATH    "...insert here..."
```